packages/app/src/runner/unifiedRunner.ts

```diff
@@ -27,6 +27,24 @@ export function useUnifiedRunner () {
       const route = useRoute()
       const selectorPlaygroundStore = useSelectorPlaygroundStore()
 
+      watch(() => specs.value, () => {
```

Comment:
@tbiethman @lmiller1990 here's the new watcher to handle removed/renamed specs, there are a couple of new tests added to the Cy-in-Cy CT and E2E specs for this as well.